src/core/mac/link_raw.cpp

```diff
@@ -159,40 +179,37 @@ otError LinkRaw::SetShortAddress(uint16_t aShortAddress)
 
 otRadioCaps LinkRaw::GetCaps(void) const
 {
-    otRadioCaps RadioCaps = otPlatRadioGetCaps(&mInstance);
-
-    // The radio shouldn't support a capability if it is being compile
-    // time included into the raw link-layer code.
+    otRadioCaps radioCaps = mRadioCaps;
 
 #if OPENTHREAD_CONFIG_ENABLE_SOFTWARE_ACK_TIMEOUT
-    if ((RadioCaps & OT_RADIO_CAPS_ACK_TIMEOUT) == 0)
+    if ((mRadioCaps & OT_RADIO_CAPS_ACK_TIMEOUT) == 0)
```

Comment:
What is the purpose of these conditionals? Why not just set the bit whenever the build option is included?